__tests__/src/lib/ThumbnailFactory.test.js

```diff
@@ -194,7 +400,42 @@ describe('getThumbnail', () => {
         ],
         type: 'Collection',
       });
-      expect(getThumbnail(collection)).toMatchObject({ url: 'https://example.org/manifest1/thumbnail.jpg' });
+      expect(getThumbnail(collection, { maxHeight: 120 })).toEqual({ url: 'https://example.org/manifest1/thumbnail.jpg' });
+    });
+  });
+
+  describe('selectBestImageSize', () => {
```

Comment:
I really like how you were able to add some tests  around this. 👏 Because of the API change that we cannot make, is it possible to add this as an additional export so it can be tested?

Reply:
I am rather new to ES6 so I may be wrong, but I don't believe it is possible to add an additional export to ThumbnailFactory without breaking its API. E.g., if we were to do instead (in ThumbnailFactory.js)

```export { ThumbnailFactory, getBestThumbnail };```

then imports would have to change from

```import getBestThumbnail from '../../lib/ThumbnailFactory';```

to

```import { getBestThumbnail } from '../../lib/ThumbnailFactory';```

That would be a breaking change, right?

(A workaround would be to move `selectBestImageSize` to a different module.)

Reply:
Could something like this work?

`export { getBestThumbnail as default, ThumbnailFactory };` 

That way, the default export does not change (the API) but you can provide additional exports

Reply:
Getting back to this now -- it looks like this works! Will do.